### ISSUE TITLE
High accuracy but low confidence

### ISSUE THREAD

**Issue by athrunsunny** [ultralytics/yolov5#9968]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

![F1_curve](https://user-images.githubusercontent.com/43680123/198829788-840b1505-68f8-4f9e-b97c-9c5247f58d8b.png)
![P_curve](https://user-images.githubusercontent.com/43680123/198829811-b77e05f8-2a94-4ec2-a07d-52d96ebd9761.png)
![R_curve](https://user-images.githubusercontent.com/43680123/198829816-e4b2de0a-598d-444f-8b8e-00d8c97d73cd.png)

    
  As shown in the figure above,use default setting to train one class custom datasets,the training results were correct at the beginning,confidence is above 0.9,after several times of training, the situation in the above figure appears,F1 and R  curve converges at about 0.5.At inference time,the highest confidence is only 0.5.And this problem will occur in later training.With the same datasets and the same settings, except for the image size(640 or 800).I don't know if this is a bug,can you help me look at this problem.

**Comment by glenn-jocher** [comment]
@athrunsunny low confidence is an indicator that you haven't trained long enough. I'd train 2x or 4x the epochs and observe your confidences then.

**Comment by athrunsunny** [comment]
It may work ,but i have another question.It's very strange. In the first few times(about 15), the results of only 300 epochs of training were in line with expectations, and the highest inference confidence was about 0.93.But from the 16th training, suddenly, the highest confidence was only 0.5,the result after that is all like this

**Comment by glenn-jocher** [comment]
@athrunsunny Strange. It’s possible that you’ve overfit to your validation set. Try removing or replacing your validation set to refresh your training.
